impl/src/main/java/com/sun/faces/context/flash/ELFlash.java

```diff
@@ -156,7 +156,7 @@ public class ELFlash extends Flash {
      */
     static final String FLASH_NOW_REQUEST_KEY = FLASH_ATTRIBUTE_NAME + "n";
 
-    private enum CONSTANTS {
+    enum CONSTANTS {
```

Comment:
Why is private being removed?

Reply:
So it's accessible from ELFlashTest in same package.

Reply:
As I don't remember so I am just going to ask. Would this impact the signature tests?

Reply:
No idea. How do I run them so I can verify?